test/math/kd_tree_test.cc

```diff
@@ -221,12 +334,12 @@ class KDTreeExtendedTest : public ::testing::Test {
   }
   // Returns a vector of @p number_of_points UniquePoints obtained randomly, with a uniform distribution, within a range
   // delimited by @p range_min and @p range_max
-  static std::vector<UniquePoint> GetRandomPoints(int number_of_points, double range_min, double range_max) {
+  static std::deque<UniquePoint> GetRandomPoints(int number_of_points, double range_min, double range_max) {
     std::random_device rd;                                               // obtain a random number from hardware
     std::mt19937 gen(rd());                                              // seed the generator
     std::uniform_real_distribution<double> distr(range_min, range_max);  // define the range
 
-    std::vector<UniquePoint> points;
+    std::deque<UniquePoint> points;
     for (int j = 0; j < number_of_points; ++j) {
       points.push_back({distr(gen), distr(gen), distr(gen), GetUniqueId()});
     }
```

Comment:
I missed this in previous reviews. Can we use the same seed always?

Reply:
Done.